### ISSUE TITLE
wazapp stop sync new contacts

### ISSUE THREAD

**Issue by Sherif78** [tgalal/wazapp#309]
Dears, 

my wazapp stop sync new contacts, I think the reason is the delay in the renewal of subscription but when i renew my subscription the problem was still.

Is there a solution?

**Comment by CODeRUS** [comment]
search first?
https://github.com/tgalal/wazapp/issues/305

**Comment by Sherif78** [comment]
yes i searched in #305 also i uninstall wazapp and delete its folder from home/user/wazapp then i reinstall it again but in vain. 
Is there a solution to this problem?

**Comment by CODeRUS** [comment]
sure. contribute to wauth-2 protocol implementation fot wazapp :)

**Comment by Sherif78** [comment]
sorry i don't understand
what is wauth-2 ?

**Comment by CODeRUS** [comment]
whatsapp protocol. nevernind.

**Comment by Sherif78** [comment]
tell me exactly what should i do with this protocol

**Comment by CODeRUS** [comment]
became a programmer

**Comment by Sherif78** [comment]
:)

**Comment by Sherif78** [comment]
Okay, when you reach to the solution to this problem please don't forget me
because I do not have enough money to buy new android mobile now :))

**Comment by CODeRUS** [comment]
you can use whatsup or persecute apps instead of wazapp for now

**Comment by Sherif78** [comment]
Great, How can I get them?

**Comment by CODeRUS** [comment]
openrepos.net

**Comment by Sherif78** [comment]
@CODeRUS  Thank you very much for your advice, i downloaded whatsup and it working properly without any error
